adapter/parser/src/main/java/org/mobilitydata/gtfsvalidator/parser/GtfsEntityParser.java

```diff
@@ -185,6 +185,28 @@ public ParsedEntity parse(RawEntity toParse) {
                                 ));
                     }
 
+                /*
+                } else if (columnSpecProto.getType().getType() ==
```

Comment:
@fabrice-v @barbeau @lionel-nj Implementing time conversion here requires to remove time type validation from `GtfsTypeValidator`, and to implement validation here, which would breaks the architecture (validation done by the parser instead of the validator).

If we don't remove the validation from the `GtfsTypeValidator` and keep the conversion in the parser, we get an exception, which is why this code is commented at the moment.

Any suggestion on the implementation? I thought maybe keep time as a string and an Util class would be the best option after all.

Reply:
As discussed, a Utility class will be implemented for time conversions. See https://github.com/MobilityData/gtfs-validator/pull/295.